src/parse/asp/objects.go

```diff
@@ -835,7 +835,11 @@ func (c *pyConfig) MarshalJSON() ([]byte, error) {
 }
 
 func (c *pyConfig) String() string {
-	return "<global config object>"
+	marshalledConfig, err := c.MarshalJSON()
+	if err != nil {
+		panic(fmt.Sprintf("failed to marshal object: %s", err))
+	}
+	return fmt.Sprintf("GlobalConfig%s", string(marshalledConfig))
 }
```

Comment:
This is quite different to `pyDict` which uses this formatting:
```
func (d pyDict) String() string {
	var b strings.Builder
	b.WriteByte('{')
	started := false
	for _, k := range d.Keys() {
		if started {
			b.WriteString(", ")
		}
		started = true
		b.WriteByte('"')
		b.WriteString(k)
		b.WriteString(`": `)
		b.WriteString(d[k].String())
	}
	b.WriteByte('}')
	return b.String()
}
```

I feel like these two should behave the same.

Reply:
Fair point! Thanks, I have addressed that.